src/excmds.ts

```diff
@@ -751,6 +751,26 @@ export async function source_quiet(...fileArr: string[]) {
     }
 }
 
+/** Use tridactylrc located at the given url
+ * @param url the url where the raw version of tridactylrc can be found
+ */
+//#background
+export async function source_from_url(url: string) {
+    if (!url) return
+    await rc.sourceFromUrl(url)
```

Comment:
Could you make sure we do some helpful stuff to turn URL-alikes to valid URLs? So we can leave off the schema, e.g. gist.github.com...

Reply:
Sorry I overlooked this. Will fix this.

Reply:
You might want to look at what we do with `open` but obviously don't fall back to running a search query.